src/Sylius/Component/Promotion/Event/CatalogPromotionUpdated.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Promotion\Event;
+
+final class CatalogPromotionUpdated
```

Comment:
```suggestion
class CatalogPromotionUpdated
```
as other events in Sylius are not final to make them easier to customize